contentcuration/contentcuration/static/js/edit_channel/models.js

```diff
@@ -470,6 +470,33 @@ var ChannelModel = BaseModel.extend({
             });
         });
         return promise;
+    },
+    activate_channel:function(){
```

Comment:
This function could return here at line 476. `return new Promise(...)` to save creating a temporary variable.

Also, would it be preferable to simply use the Deferrable that is returned by `$.ajax` instead of wrapping it in a Promise?